eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/ExceptionThrownTest.java

```diff
@@ -30,12 +28,12 @@ public void specificRuntimeException()
     {
         try
         {
-            assertThrows(NullPointerException.class, new EmptyRunnable());
+            assertThrows(NullPointerException.class, EmptyRunnable::new);
```

Comment:
This implies we have a source breaking change?

Reply:
This call used to be based on the Junit4 assertThrows method, now it's based on Junit5. The signature of the method changes from Junit4 to 5 so as a result of the migration we had to change that.

Reply:
This has the same "problem" as the code below. It's no longer a runnable that does nothing, it's now a runnable that creates a runnable. This isn't really a problem on its own, but it no longer proves that Verify type signatures are source compatible with the previous version.

Reply:
What's your advice then?